### ISSUE TITLE
Modeless dialog boxes needs keyboard access

### ISSUE THREAD

**Issue by henrik-jensen** [microsoft/vscode#134039]
Feature request in short: Access with keyboard to modeless dialog boxes.

Below is a screenshot from a vanilla portable/zip version (`VSCode-win32-x64-1.60.2.zip`) to illustrate my point.
I have loaded a file that the OS has marked as readonly and now trying to save it with [CTRL+S]:

![02_saving_readonly_file](https://user-images.githubusercontent.com/9413242/135135008-97b1bb07-1e3c-49f6-9244-d8063bf3bce7.png)

The only keyboard interaction with this message box, "Failed to save...",  is [Escape] which cancels the *save* action and the modeless dialog, otherwise you have to use the ~~rat~~ mouse to interact with this window.

It would be nice to be able to navigate to these modeless dialog boxes to make a choice without  using the mouse.

PS. This could easily be categorized as an accessibility *bug*, but I'll give you some slack and place it under *Feature request*, because I think I'm an [Alright Guy](https://www.youtube.com/watch?v=dXL_ZTaA6p4) and [Seattle](https://www.youtube.com/watch?v=CZWuDkHLBLo) do seem to be such a nice place. 😋

**Comment by bpasero** [comment]
Notification toasts are fully keyboard accessible.

**Comment by henrik-jensen** [comment]
@bpasero:
> Notification toasts are fully keyboard accessible.

Ahh,- A non-modal message dialog is a "Notification toasts" in vscode lingo.
`View->Command Palette->Notifications: Show Notifications` solved it.
Alternatively creating a keybinding to `notifications.focusToast`.

Thanks.
